### DISCUSSION TITLE
Building on top of FastF1 with AI

### DISCUSSION THREAD

**Discussion by Draichi** [theOehrly/Fast-F1#598] [Show and tell]
Hello FastF1 Community,

Firstly, FastF1 is a fantastic project and I appreciate the efforts of everyone involved! 👏

I am thrilled to share an exciting new open-source project I've been working on: [fastf1-predictions](https://github.com/Draichi/fastf1-predictions). I built this project on top of the FastF1 library to bring the power of artificial intelligence to racing analysis.

The project currently features a lap time predictor that uses AI to analyze telemetry data and forecast a car's performance during a race. The next phase involves developing an AI assistant to help optimize tire strategies for different circuits, providing crucial insights to drivers and racing engineers.

I invite everyone in the community to explore the project, contribute, and share your feedback. Your expertise and collaboration can help us refine these tools and drive innovation in motorsport analytics.

**Comment by DasMechaniker** [comment]
What kind of tire models are you using? And what simulation stratergy, QSS or QTS? Where are you getting technical data for tire grip coefficients or vehicle coefficients of drag and lift?
